python/phevaluator/evaluator_omaha.py

```diff
@@ -12,6 +12,12 @@
 
 COMMUNITY_CARD_COUNT = 5
 HOLE_CARD_COUNT = 4
+FLUSH_BASE_VALUE = 10000
+NO_FLUSH_BASE_VALUE = 10000
```

Comment:
```suggestion
MAX_RANK_VALUE = 10000
```
Sorry, I overlooked this one. We can merge these two declarations as one constant.

Reply:
Done